Tasks/AzureFunctionAppV1/task.json

```diff
@@ -116,10 +116,20 @@
                 "EditableOptions": "True"
             },
             "options": {
-                "DOCKER|microsoft/azure-functions-dotnet-core2.0:2.0": ".NET",
-                "DOCKER|microsoft/azure-functions-node8:2.0": "JavaScript"
+                "DOTNET|2.2": "DOTNET|2.2 (functionapp v2)",
```

Comment:
this will reset existing selection of users. can you check in telemetry for how many users this field is set?

Reply:
This will not change the existing selection of the user as this field in editable and can be filled with any value that the user desires.

Reply:
Here dropdown values are in the format of key value, where value is used to update `FUNCTIONS_WORKER_RUNTIME` while key is used to update `linuxFxVersion`. 

if we assume that existing selection of user from `.Net` or `Javascript` remains as is due to editable option.
this will put `linuxFxVersion` with bad value.

and `FUNCTIONS_WORKER_RUNTIME` will not be updated due to this If condition https://github.com/microsoft/azure-pipelines-tasks/blob/a370f6ab16aade62e2f4d0d82a532ea88c8e607b/Tasks/AzureFunctionAppV1/deploymentProvider/BuiltInLinuxWebAppDeploymentProvider.ts#L32

So I think user has to open pipeline once and save it with proper value.

and if this assumption is correct then we need to check how many users are impacted (I think it will be minimal, as existing values were putting app in bad shape and we had advised to remove those value from selection)

Reply:
The existing value would appear as:
![image](https://user-images.githubusercontent.com/8944724/103028023-f85c6c80-457c-11eb-9438-86d4c3d4362d.png)

I've left the old values as is in the linuxFunctionRuntimeSettingValue map, so the old values won't appear in the dropdown but will keep on working as is without breaking any of the existing pipeline and if someone uses the new value from the dropdown they will have a bug free experience. So this change is kind of backward compatible.

Reply:
as far as I remember if key value is changed, RD will be shown as unsaved.

@amit-avit could you please test following
1. Create clean set-up, have two RDs - first with .Net as run time stack and other as JavaScript
2.  Run both RD ( lets have green run ) 
3. Upload the updated task with new values

Validate the RDs and runs